apps/web/app/[lang]/studies/[id]/helpers.ts

```diff
@@ -53,3 +54,39 @@ export const GenderToText = (gender: string) => {
       return t('allGender');
   }
 };
+
+export const getTypeformIds = (
+  study: Study,
+  campaignId: string,
+  countryId: string,
+) => {
+  let form: Form;
+
+  if (campaignId) {
+    form = study.forms.find((form) => form.id === campaignId);
+  }
+  if (countryId) {
+    form = study.forms.find((form) => form.country_id === countryId);
+  } else {
+    return {};
+  }
+
```

Comment:
We are not checking if the form was found actually. We must check this. Apply this suggestion
```suggestion
if (!form) return {};
```